public/js/draw/setup.js

```diff
@@ -86,19 +86,47 @@ $('#finish-region').click(function () {
     finishRegion();
     });
 
-$('#submit-gid').click(function() {
-       $.ajax({
-            url: 'get-json-data',
-            data: {gid : $('#area-of-study').val()},
-            dataType: 'json', 
-            success: function(data) {
-                $('#json-data').html('<pre>' + JSON.stringify(data) + '<pre>');
-            },
-            error: function(xhr, status, err) {
-                console.error('graphs', status, err.toString());
+$('#save-graph').click(function () {
+    $.ajax({
+        url: 'insert-graph',
```

Comment:
The only reason I separate it into two Ajax requests is because of the situation where the user enters an invalid graph name multiple times. I don't want to send the us in data until a valid name is entered.